tests/materialdata/woehler/test_analyzer.py

```diff
@@ -595,3 +595,43 @@ def test_max_likelihood_one_mixed_horizon():
         wc = ml.analyze().sort_index()
     bic = ml.bayesian_information_criterion()
     pd.testing.assert_series_equal(wc, expected, rtol=1e-1)
+
+
+def test_irrelevant_runouts_dropped():
+    fd = woehler.determine_fractures(data_pure_runout_horizon_and_mixed_horizons, 1e7).fatigue_data
+    num_data_before = len(fd._obj)
+    num_tests_lowest_pure_runout_load_level = len(fd.load[fd.load==fd.pure_runout_loads.min()])
```

Comment:
Please hard code the values of the two `num_` variables in order to make tests simpler and avoid to make the same thinko in the tests as in the production code.

Reply:
Hey Johannes,
thanks for the feed back. I added the request.
BR, Dominic